src/nvim/os/lang.c

```diff
@@ -5,18 +5,13 @@
 # define Boolean CFBoolean  // Avoid conflict with API's Boolean
 # define FileInfo CSFileInfo  // Avoid conflict with API's Fileinfo
 # include <CoreServices/CoreServices.h>
+
 # undef Boolean
 # undef FileInfo
 #endif
 
-#include "auto/config.h"
-
-#ifdef HAVE_LOCALE_H
-# include <locale.h>
-#endif
-
```

Comment:
Removing this doesn't look right.

Reply:
I'll move the includes into the `#ifdef __APPLE__` then so it's only removed if it's not needed on osx. Should probably do this for all os-dependent includes now that I think about it.